localstack/services/cloudformation/models/apigateway.py

```diff
@@ -365,26 +372,30 @@ def _subresources(resource_id, resources, resource_type, func, stack_name):
 
                     kwargs["uri"] = uri
 
-                if integration.get("IntegrationHttpMethod"):
-                    kwargs["integrationHttpMethod"] = integration["IntegrationHttpMethod"]
-
-                if integration.get("RequestTemplates"):
-                    kwargs["requestTemplates"] = integration["RequestTemplates"]
-
-                if integration.get("Credentials"):
-                    kwargs["credentials"] = integration["Credentials"]
-
-                if integration.get("RequestParameters"):
-                    kwargs["requestParameters"] = integration["RequestParameters"]
+                integration_responses = []
+                if kwargs.get("integrationResponses"):
+                    integration_responses = kwargs["integrationResponses"]
+                    del kwargs["integrationResponses"]
```

Comment:
nit: could this work? Then you don't have to don't have to check and delete from `kwargs`
```suggestion
                integration_responses = kwargs.pop("integrationResponses", [])
```